### ISSUE TITLE
Color Picker: Can we go for hex 8 digits?

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5132]
Could we jump from rgba() to hex 8 digits?

https://www.w3.org/TR/css-color-4/#hex-notation
https://caniuse.com/?search=%23RRGGBBAA
It seems pretty safe to use

<img width="194" alt="image" src="https://user-images.githubusercontent.com/7963804/194542283-c8cda460-bf7f-4f13-a660-91557e0dd5ae.png">

<img width="244" alt="image" src="https://user-images.githubusercontent.com/7963804/194542354-fa1a0734-e8c9-47e5-a64d-18ad73878921.png">

The patterns is valid and can be written, right now we change the 8 digits value to `rgba()` pattern.
If we try to change it it will be converted to hex 6 digits

<img width="285" alt="image" src="https://user-images.githubusercontent.com/7963804/194543020-e19b127a-0151-4255-916d-83b0641de23a.png">

Some product teams are currently using 8 digits so I think we could use Hex by default and rgba() when we specify by typing

**Comment by pat270** [comment]
Just leaving this here #4134.

**Comment by marcoscv-work** [comment]
Thanks guys! pretty clear, we need to add an extra alpha bar in the editor, this time to fix a bug in the portal, so we're probably more ready to prioritize it

**Comment by drakonux** [comment]
Please @matuzalemsteles if you need work from the team please open a DESIGN ticket to request something to Lexicon if you need something from us related to this. Thanks in advance.

**Comment by marcoscv-work** [comment]
Some accessibility issues have been discovered after a manual accessibility check (related to button range). Could we review/solve them in the same ticket from a design point of view? :-)

**Comment by pat270** [comment]
For the accessibility issue, we could change the hue selector to use a range input. It might solve that problem among others we don't know about yet.

**Comment by marcoscv-work** [comment]
> For the accessibility issue, we could change the hue selector to use a range input. It might solve that problem among others we don't know about yet.

That would be perfect, can you work on that? do you need an issue?

**Comment by pat270** [comment]
@marcoscv-work I think we can create a separate issue for it. I'll make one today if I don't see it.

**Comment by emiliano-cicero** [comment]
Hey guys, I've prepared this [Figma file](https://www.figma.com/file/HSKtTemAUe0HTMRW7KMqrM/DESIGN-000-color-picker-proposal?node-id=1%3A3886) with some changes to the picker.
 
I've changed the selected style, adding a check icon.
And also added 2 new inputs to manage the alpha + the hue. 
<img width="469" alt="image" src="https://user-images.githubusercontent.com/27729173/195135184-103c1416-1412-405e-b564-9a83c56c054d.png">

**Comment by matuzalemsteles** [comment]
Hey @emiliano-cicero now the color that is with the border is this only when it has focus?

**Comment by marcoscv-work** [comment]
> Hey @emiliano-cicero now the color that is with the border is this only when it has focus?

Yes, now the selected one is tabindex=-1 and marked with icon!!! and as Patrick said, best solution to ranges inputs is using native (clay) range!

**Comment by matuzalemsteles** [comment]
Yeah using the range for the thumb makes sense instead of the button! Just to confirm, are we now always going to show the alpha + matrix inputs and the transparency input visual or should we only show this when it is defined in the input?

**Comment by pat270** [comment]
@emiliano-cicero Just want to confirm if the H input is needed. Our current hue selector just updates the RGB values.

**Comment by marcoscv-work** [comment]
@matuzalemsteles! Thats true, it is enough if we make the color as disable because it is selected.

@pat270 H input should be in XX format, not 0.0 format. I think it is there because there are always input representation for a range.

**Comment by matuzalemsteles** [comment]
LPS Ticket https://liferay.atlassian.net/browse/LPS-190553
